adserver/models.py

```diff
@@ -1029,7 +1029,8 @@ def track_view(self, request, publisher, url):
         """
         self.incr(VIEWS, publisher)
 
-        if settings.ADSERVER_RECORD_VIEWS:
+        # TODO: Find a better way to record explicit ad server views
+        if settings.ADSERVER_RECORD_VIEWS or "readthedocs" not in publisher.slug:
```

Comment:
I do want to note that I view this as temporary and we will need a better solution. This sort of breaks the setting for anyone else who uses our ad server.

Reply:
Totally -- I think an explicit flag on the publisher is probably the correct option.

Reply:
I think that's a better option, overall. Maybe we have the global setting and it we have a publisher specific setting that is default=False.